packages/components/src/utils/unit-values.ts

```diff
@@ -10,7 +10,7 @@ const UNITED_VALUE_REGEX =
  */
 export function parseCSSUnitValue(
 	toParse: string
-): [ number | undefined, string | undefined ] {
+): [ number | undefined , string | undefined  ] {
```

Comment:
Seems like it's adding unwanted extra spaces in tuples, regarding of whether they're on multiple lines or not.

Reply:
Union types are fixed :+1: